Pipfile

```diff
@@ -38,7 +38,7 @@ pytest-mock = "*"
 
 [packages]
 colorama = "*"
-flask = "*"
+flask = "==2.2.2"
```

Comment:
If not already done, can you raise a card to unpin and update all packages.

Reply:
Will add a card, was planning to when it merged 👍